src/test/test_snap_mapper.cc

```diff
@@ -480,9 +491,31 @@ class MapperVerifier {
     }
   }
 
-  void create_snap() {
-    snap_to_hobject[next];
+  snapid_t create_snap() {
+    snapid_t snapid = next;
+    snap_to_hobject[snapid];
     ++next;
+
+    return snapid;
+  }
+
+  // must be called with lock held to protect access to
+  // hobject_to_snap and snap_to_hobject
+  void __add_object(const hobject_t & obj, const set<snapid_t> &snaps) {
```

Comment:
Identifiers with two initial underscores or an initial underscore followed by an uppercase letter are reserved globally for use by the compiler.

Reply:
Fully agreed. [It's an old restriction](https://stackoverflow.com/a/224420), let's to break it.